frontend/src/container/InfraMonitoringHosts/utils.tsx

```diff
@@ -164,6 +181,22 @@ export const formatDataForTable = (data: HostData[]): HostRowData[] =>
 		),
 		wait: `${Number((host.wait * 100).toFixed(1))}%`,
 		load15: host.load15,
+		diskUsage: (
+			<div className="progress-container">
+				<Progress
+					percent={Number((host.diskUsage * 100).toFixed(1))}
+					strokeLinecap="butt"
+					size="small"
+					strokeColor={((): string => {
+						const diskUsagePercent = Number((host.diskUsage * 100).toFixed(1));
```

Comment:
Please let me know if the colour distribution needs to be changed

Reply:
This is fine.